pkg/power/uptime.go

```diff
@@ -129,5 +129,9 @@ func isNodeHealthy() bool {
 		log.Error().Msg("node is running on limited cache")
 		healthy = false
 	}
+	if app.CheckFlag(app.NotReachable) {
```

Comment:
Why not invoke the Network health task here and if it returns an error you return false?

Reply:
preferred to be aligned with the other health check, and make the uptime only check the flags. ok?